recipes/Clp/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "coin-or-clp" %}
+{% set version = "1.17.4" %}
```

Comment:
Latest is 1.17.6

Reply:
google or-tools seems to be still pinning 1.17.4.
Last time I tried a test in or-tools was failing with 1.17.5 as far as I remember. We'll have all versions eventually (the conda-forge bots usually do this for us automatically) but let's start with this one please :)